src/sql/workbench/contrib/assessment/browser/consts.ts

```diff
@@ -0,0 +1,16 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+export enum AssessmentTargetType {
+	Server = 1,
+	Database = 2
+}
+
+export enum AssessmentType {
+	AvailableRules = 1,
+	InvokeAssessment = 2
+}
+
+export const TELEMETRY_VIEW_EVENT = 'Asmt.View';
```

Comment:
We already have a view event that takes in an area name (likely just ASMT for your purposes).

If you're truly logging actions then you should be giving them better names.

Let me know if you have further questions on the telemetry

Reply:
Ok. I named it AssessmentTab, so actions will be logged with 'AssessmentTab' as a view and 'asmtaction.server.getitems' as an action name.